staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -327,9 +327,6 @@ const (
 	// Reflects whether the QEMU guest agent updated access credentials successfully
 	VirtualMachineInstanceAccessCredentialsSynchronized VirtualMachineInstanceConditionType = "AccessCredentialsSynchronized"
 
-	// Reflects whether the QEMU guest agent is connected through the channel
-	VirtualMachineInstanceUnsupportedAgent VirtualMachineInstanceConditionType = "AgentVersionNotSupported"
```

Comment:
@jelkosz Heads up that I've pulled this value out of types.go. Just double checking that no other projects use it directly.